crates/holochain/src/local_network_tests.rs

```diff
@@ -71,7 +71,9 @@ fn conductors_call_remote(num_conductors: usize) {
 
         let mut envs = Vec::with_capacity(handles.len());
         for h in &handles {
-            envs.push(h.get_p2p_env().await);
+            let space = h.cell_id.dna_hash().get_raw_36().to_vec();
+            let space = Arc::new(kitsune_p2p::KitsuneSpace(space));
```

Comment:
Could be nice to `impl From<DnaHash> for KitsuneSpace`

Reply:
Alas, neither of these types can know about each other... holo_hash because it is used in zome_types, and kitsune because it is holochain agnostic...

I suppose i could make an extension trait somewhere...

Reply:
ah. maybe just a function then?

Reply:
there were some `pub(crate)` extension traits in holochain_p2p that did exactly this... so I made them `pub` and implemented using them :+1: